### PR TITLE
add tutorial to install apache2 with letsencrypt

### PR DESCRIPTION
**Pull request by sap-dev**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Stefan Dunkel <stefan@stefan6.de>

Please use stefan@stefan6.de for communication, i dont can change the github email. Thanks.

### CODE REVIEW

File: tutorials/install-apache2-and-letsencrpt/01.en.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+path: "/tutorials/install-apache2-and-letsancrypt"
+slug: "install-apache2-and-letsancrypt"
+date: "2019-03-21"
+title: "Install Apache2 with LetsEncrypt SSL:"
+short_description: "Install Apache2 with an LetsEncrpy Cert on a new Server"
+tags: ["Development", "Lang:Bash"]
+author: "Stefan Dunkel"
+author_link: "https://github.com/sap-dev"
+language: "en"
+available_languages: [""]
```
Comment (n-se) [inline tutorials/install-apache2-and-letsencrpt/01.en.md]:
```suggestion
available_languages: ["en"]
```

**Comment by n-se** [comment]
Please always use only one ` in front of and behind if you do not have a codeblock and just some single words.
Also do not use `test.com`, always use `example.com` or a subdomain `test.example.com`. Anyway you should use in this tutorial `example.com`, a subdomain is not needed here.

In this tutorial is a lot of stuff missing, you do not explain how to test the SSL cert. Also users need to setup proper DNS records otherwise certbot will fail. Please explain more detailed what a user has to do. You should also somewhere note where the Let's Encrypt certificates are located.

**Comment by n-se** [comment]
Hi @sap-dev, do you still wanna work on this tutorial? @1817666 plans to create a similar tutorial, so we would like to hear some feedback from you.

**Comment by leojakeson** [comment]
No longer being worked on.

**Pull request closed without merging**